liblxqt-config-cursor/man/lxqt-config-input.1

```diff
@@ -0,0 +1,29 @@
+.TH lxqt-config-input "1" "January 2025" "LXQt 2.1.0" "LXQt Input Settings"
```

Comment:
Change `lxqt-config-input` to uppercase (I see that for some reason headers needs that to display bold and colored). Needed for all other man pages too.